crates/loader/src/bindle/deprecation.rs

```diff
@@ -0,0 +1,17 @@
+use once_cell::sync::OnceCell;
+
+static PRINTED: OnceCell<bool> = OnceCell::new();
+
+/// Prints a notice that Bindle support is deprecated.
+pub fn print_bindle_deprecation() {
+    if PRINTED.get().is_some() {
+        return;
+    }
+
+    eprintln!("WARNING: Bindle support is deprecated and will be removed in a future version.");
+    eprintln!("For remote applications, please use registry commands and features instead.");
+    eprintln!("See https://developer.fermyon.com/spin/spin-oci for more details");
+    eprintln!();
+
+    _ = PRINTED.set(true);
+}
```

Comment:
Stdlib has `Once`:

```suggestion
use std::sync::Once;

/// Prints a notice that Bindle support is deprecated.
pub fn print_bindle_deprecation() {
    static PRINT: Once = Once::new();
    PRINT.call_once(|| {
        eprintln!("WARNING: Bindle support is deprecated and will be removed in a future version.");
        eprintln!("For remote applications, please use registry commands and features instead.");
        eprintln!("See https://developer.fermyon.com/spin/spin-oci for more details");
        eprintln!();
    });
}
```

Reply:
Thanks!  I thought there was something in stdlib but found only its `OnceCell` which was nightly only...